backend/qemu.pm

```diff
@@ -738,26 +701,29 @@ sub start_qemu {
             if ($vars->{VNC} !~ /:/) {
                 $vars->{VNC} = ":$vars->{VNC}";
             }
-            push(@params, "-vnc", "$vars->{VNC},share=force-shared");
-            push(@params, "-k", $vars->{VNCKB}) if ($vars->{VNCKB});
+            gen_params @params, 'vnc', "$vars->{VNC},share=force-shared";
+            gen_params @params, 'k',   $vars->{VNCKB};
```

Comment:
This regards just coverage actually, `gen_params` checks if the variable is empty, and don't push it into the array in that case.